Design.md

```diff
@@ -0,0 +1,52 @@
+# Design
+ 
+## Senior UI/UX Designer
+- I have at least 5 years' experience
+- I display initiative and ownership
+
+- I can lead a project day to day (Design Lead)
+    - I can make strategic design and UX decisions about both core and new functions and features
+    - I can measure and capture user feedback on a regular basis.
+- I can create design guidelines to set the tone and design rules through out a project life cycle
+- I can oversee the user experience from conception to launch in partnership with Product Owners and developers
+- I am responsible for overall team performance and building a culture of design excellence
+- I am comfortable with running the Scrum ceremonies (i.e. can act as Scrum Master)
+- I am comfortable with mentoring/teaching other designers 
+- I can define the user experience best practice for client porjects and internal projects
+
+
+##  UI/UX Designer
+- I have at least 3 years' experience
+- I have excellent visual design skills with an eye for detail
+
+- I can build/gather requirements from client conversations
+    - I can assist on Specification Reviews
+    - I can do initial meetings with clients
+    - I can create the design of user interface specifications, by applying client research and design guidelines
+- I can design user interfaces that encompass an end-to-end experience across a variety of platforms
+- I can represent work to clients and other team members, clearly and succinctly articulating the goals and design concepts
+- I can excute the UX process from end-to-end, including wireframes, behavioral specifications, and high-fidelity prototypes
+- I take collaborative approach to working with engineering and product teams
+- I am comfortable with all facets of the Scrum process 
+    - I provide suggestions in Scrum Reviews/Retros
+    - I break PBIs down in planning
+    - I can estimate work to be done
+    - I help the Product Owner and Scrum Master proactively (i.e. give suggestions/recommendations)
+- I provide effective feedback internally
+
+
+## Design Work Experience
+- I have approximately 1 year's of experience in graphic or user interface design (in industry or otherwise)
+- or I may be just out of University and/or still studying for a design related degree
```

Comment:
1. Work experience requiring 1 year experience is odd - I think we should remove this.
2. Line 40 here is strangely worded, should instead say something like "I have a demonstrated interest in the field" for example.

Reply:
1. >Work experience requiring 1 year's experience is odd - I think we should remove this.
Done. 

2. >Line 40 here is strangely worded, should instead say something like "I have a demonstrated interest in the field" for example.
It's similar with what we have for devs. I guess we should keep this, so we have something related to education. 
I also added a line "I have a demonstrated interest in the field of graphic desigan, interface design or digital media"